common-util/src/test/java/org/sunbird/services/sso/impl/KeyCloakRsaKeyFetcherTest.java

```diff
@@ -0,0 +1,23 @@
+package org.sunbird.services.sso.impl;
+
+import java.security.PublicKey;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.sunbird.common.models.util.KeyCloakConnectionProvider;
+
+/**
+ * @author   github.com/iostream04
+ *  This class will Test KeyCloakRsaKeyFetcherTest's getPublicKeyFromKeyCloak method .
+ */
+
+public class KeyCloakRsaKeyFetcherTest {
+
+  @Test
+  public void getPublicKeyFromKeyCloakTest() {
+    PublicKey key = new KeyCloakRsaKeyFetcher().getPublicKeyFromKeyCloak(KeyCloakConnectionProvider.SSO_URL,
```

Comment:
Unit tests shouldn't rely on servers being up for the tests to pass. You can use power mockito framework to mock dependencies like Cassandra, ES, Keycloak etc.

Reply:
e.g. SystemSettingsActorTest.java

Reply:
Thank you